config/model_configs/diagnostic_edmfx_rico_box.yml

```diff
@@ -8,13 +8,15 @@ implicit_diffusion: true
 approximate_linear_solve_iters: 2
 prognostic_tke: true
 edmfx_upwinding: first_order
-edmfx_entr_model: "Generalized" 
-edmfx_detr_model: "Generalized" 
+edmfx_entr_model: "Generalized"
+edmfx_detr_model: "Generalized"
 edmfx_nh_pressure: true
 edmfx_sgs_mass_flux: true
 edmfx_sgs_diffusive_flux: true
 moist: equil
-precip_model: "0M"
+apply_limiter: false
+precip_model: "1M"
+override_τ_precip: false
```

Comment:
Does `override_τ_precip` do anything when using 1-moment?

Reply:
No, it doesn't. This overrides the 0M precipitation formation timescale, so it is not used in 1M simulations. I'll delete it from the config.